### DISCUSSION TITLE
Can't build project after installing swift-composable-architecture using SPM. Seeking help to setup.

### DISCUSSION THREAD

**Discussion by Shayokh144** [pointfreeco/swift-composable-architecture#3511] [Q&A] [answered]
### Steps:
- Created an `SwiftUI` app 
- Build and run successfully
- Then add `swift-composable-architecture` using `SPM`
- Getting build error in `PerceptionCore` module
<img width="1382" alt="Screenshot 2024-11-27 at 2 00 58 PM" src="https://github.com/user-attachments/assets/b93c468e-3bfd-4e2d-834e-727e05b7275e">

### Specs:
- Xcode 15.2
- Min iOS version: 16.0
- Swift 5

### Error:
<img width="1403" alt="Screenshot 2024-11-27 at 3 33 11 PM" src="https://github.com/user-attachments/assets/2c16f5ab-2117-4eb4-8517-c77aa97a9b81">

Please let me know if you have any idea about how to fix this.

**Comment by mbrandonw** [comment]
Hi @Shayokh144, we have a fix for that we will merge and release soon. In the meantime you could update to Xcode 15.4+ to fix the problem.

**Reply by Shayokh144** [reply]
thanks a lot for the answer, will try to update Xcode in this weekend. I will come back here once I have some update.

**Reply by mbrandonw** [reply, marked as answer]
A fix was also released so you can continue using 15.2 for the time being.

**Reply by Shayokh144** [reply]
@mbrandonw it's working now in Xcode 15.2.. Thanks a lot
